contrib/completions/bash/s2i

```diff
@@ -324,6 +324,8 @@ _s2i_build()
     local_nonpersistent_flags+=("--volume=")
     flags+=("--ca=")
     flags+=("--cert=")
+    flags+=("--container-manager=")
```

Comment:
names can be hard .... coming up with something that captures what `docker` vs. `buildah` vs. the other tools in this space certainly falls in that category

I'm pondering the use of `container-manager` .... perhaps "safe" but feels overreaching to me

My best alternative is `oci-tool`  ... we are talking about producing OCI images, right?

Reply:
In the community we see Docker being referred as a "container-manager", and to complete the task of building an container image requires traits of the "container-manager". And, indeed, the term is overreaching, can imply a number of different tools.

So, maybe we would like to have the creation of OCI images as a flag, and therefore, we could add `--oci` boolean flag to turn on this behavior. What do you think about enabling this as a flag, instead of a string parameter?

Reply:
Yeah regardless of how Docker may have referred to itself upstream (and I suspect overreaching may have been purposeful), I like `--oci` better.

Reply:
Okay, I open to have it as a flag. @sbose78, @isutton, what's your take on this?

Reply:
If having it as a flag is merely deciding one or the other, and the choices are Docker (non OCI) and Buildah (OCI), I'm fine with a boolean flag to enable or disable it in the CLI domain.

Reply:
so is `--oci` meant to enable buildah, and the default would still be docker? I think this is confusing. After all Docker has some kind of relationship to OCI too, and that also could evolve as time goes on. `--container-manager` is more explicit if overreaching. Also what if a third (or fourth) option is added later? E.g. Podman — `--container-manager` is not a binary switch at least.